libs/platform/src/lib/components/form/radio-group/radio/radio.component.html

```diff
@@ -3,9 +3,17 @@
         [id]="id"
         [name]="name"
         [value]="value"
-        [state]="state"
+        [state]="stateType"
         [compact]="contentDensity === 'compact'"
         [disabled]="disabled"
+        [tabIndex]="isChecked ? 0 : tabIndex"
+        [title]="title"
+        [aria-label]="ariaLabel"
+        [aria-labelledby]="ariaLabelledby"
+        [aria-describedby]="ariaDescribedby"
+        [aria-disabled]="ariaDisabled"
```

Comment:
Should this be `[attr.aria-label]`, `[attr.aria-labelledby]`, etc.?

Reply:
i have used [attr.aria-label], [attr.aria-labelledby] in core/radio.Here from platform radio i am passing these values to core and core is setting these values to native <input type="radio"> as [attr.aria-label], [attr.aria-labelledby].

Reply:
Does core have `aria-label` as an `@Input`? I don't see it in the code:

https://github.com/SAP/fundamental-ngx/blob/master/libs/core/src/lib/radio/radio-button/radio-button.component.ts

Reply:
This PR includes core/radio/radion-button.component.ts file as well. here i have included " aria-label", "aria-labelledby" as Input().

Reply:
I see now. I missed that part of the PR.